example/kld.toml

```diff
@@ -141,4 +147,10 @@ public_ssh_keys = ["ssh-ed25519 AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA..
 # The http basic auth password to access self monitoring server
 # self_monitoring_password = ""
 
+# The communication port of kld
+# kld_rest_api_port = 2244
+
+# Expose communication port of kld when extenal using kld-cli
+# kld_expose_rest_api = false
```

Comment:
@JosephGoulden do you feel comfortable exposing the REST api to the internet, authenticated by macaroon? If not there might be a way using an ssh tunnel, which we also use to exposing the neard debugging dashboard to the user. That option also would avoid having to adding ip addresses to certificates and updating them if they change.

Reply:
I think first question is "do you want to run kld-cli remotely?". Because if not, just leave this as false. I will make an issue for the ssh proxy as I think it's also a good option to have available.

Reply:
We already have it... https://github.com/kuutamolabs/lightning-knd/issues/258

Reply:
No I'm not comfortable with it and would not use this option, I prefer SSH tunnel. @paulmattei wanted it. Depends on risk/usability tradeoff I suppose.

Reply:
We are using open api here.
https://github.com/kuutamolabs/lightning-knd/blob/main/kld/src/api/spec.yaml

There are trade-offs when using open API, for example, the type is less and not precise. 

Why do we use open API here if the kld rest API is not externally used?